src/seedu/addressbook/common/Utils.java

```diff
@@ -54,6 +54,10 @@ private static boolean objectIsInList (List<Object> list, Object object) {
      */
     public static boolean isValidDate(String value) {
         boolean valid;
+        if ("0".equals(value)) {
+            valid = true;
+            return valid;
+        }
```

Comment:
This seems.... very excessive usage of variables. Copy pasta for life

Reply:
I've removed this and added the check for 0 in all usage in attendance commands. The reason is that 0 is not a valid date logically.